packages/server/src/database/knex/migrations/20200425004140_create-admins.js

```diff
@@ -0,0 +1,23 @@
+
+exports.up = function(knex) {
+  return knex.schema.hasTable('Admins').then(function(sequelizeExists) {
+    if (sequelizeExists) {
+      return knex.schema.alterTable('Admins', function (table) {
+        table.datetime('createdAt').defaultTo(knex.fn.now()).alter()
+        table.datetime('updatedAt').defaultTo(knex.fn.now()).alter()
+      })
+    } else {
+      return knex.schema.createTable('Admins', function (table) {
+        table.increments('id')
+        table.string('email').unique()
```

Comment:
It would be useful to have an index here, not sure if `unique` does it by default

Reply:
Yep unique is implemented by adding an index under the hood: https://www.postgresql.org/docs/9.4/ddl-constraints.html